packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -248,7 +246,6 @@ contract PerpetualPositionManager is FundingRatePayer, FundingRateApplier {
         public
         notEmergencyShutdown()
         noPendingWithdrawal(msg.sender)
-        updateFundingRate()
```

Comment:
Same question as for `cancelWithdrawal`, are you just making this consistent with it not changing the fee-multiplier?

Reply:
Yep, exactly.